smt/div_rem_i64.py

```diff
@@ -0,0 +1,67 @@
+from z3 import *
+from utils import *
+from functools import reduce
+
+s = init_z3_solver()
+
+# define spec
+lhs, rhs, res_d, res_m = Ints('lhs rhs res_d res_m')
+
+wasm_div_i64 = Function('WasmDivI64', IntSort(), IntSort(), IntSort())
```

Comment:
There are div_u and div_s in wasm. I think this spec is div_u only, please rename this function.